packages/optimistic-governor/src/mappings/optimisticGovernor.ts

```diff
@@ -40,6 +40,9 @@ function getMasterCopy(_network: string): string {
   if (_network == "goerli") {
     return "0x07a7Be7AA4AaD42696A17e974486cb64A4daC47b";
   }
+  if (_network == "core") {
```

Comment:
coredao is running separate node instances for their mainnet and testnet subgraphs, but both of them name network as `core`. This means we have a conflicting setup and will need to change this address once deploying to Core mainnet